packages/starknet-snap/snap.config.ts

```diff
@@ -15,6 +15,7 @@ const config: SnapConfig = {
     SNAP_ENV: process.env.SNAP_ENV ?? 'prod',
     VOYAGER_API_KEY: process.env.VOYAGER_API_KEY ?? '',
```

Comment:
When will we discontinue VOYAGER ?

Reply:
until the get-stransactions fully released , then we are safe to remove